Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.2.0
+- Adds web-based reCAPTCHA fallback for phone number auth when APNs is unavailable.
```

Comment:
- Adds new phone number verification API which makes use of an intelligent reCAPTCHA to verify the application.

Reply:
Done.